app/server.go

```diff
@@ -1020,8 +1020,7 @@ func (s *Server) Start() error {
 
 	listener, err := net.Listen("tcp", addr)
 	if err != nil {
-		errors.Wrapf(err, utils.T("api.server.start_server.starting.critical"), err)
-		return err
+		return errors.Wrapf(err, utils.T("api.server.start_server.starting.critical"), err)
```

Comment:
Should we also change `api.server.start_server.starting.critical` to not use `%v` on translated text? Right now the code below prints duplicated error message.
```golang
err := errors.New("some error")
wErr := errors.Wrapf(err, utils.T("api.server.start_server.starting.critical"), err)
fmt.Println(wErr.Error())
```
output:
`Error starting server, err:some error: some error`

Alternatively, we can use `fmt.Errorf(utils.T("api.server.start_server.starting.critical"), err)` and change `%v` to `%w` in the translated text.

Reply:
Created a [ticket](https://mattermost.atlassian.net/browse/MM-32274) for this 👍